src/time_integration/paired_explicit_runge_kutta/methods_PERK4.jl

```diff
@@ -0,0 +1,326 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Compute the Butcher tableau for a paired explicit Runge-Kutta method order 4
+# using a list of eigenvalues
+function compute_PairedExplicitRK4_butcher_tableau(num_stages, tspan,
+                                                   eig_vals::Vector{ComplexF64};
+                                                   verbose = false, cS3)
+    c = ones(num_stages) # Best internal stability properties
+    c[1] = 0.0
+
+    c[num_stages - 3] = cS3
+    c[num_stages - 2] = 0.479274057836310
+    c[num_stages - 1] = sqrt(3) / 6 + 0.5
+    c[num_stages] = -sqrt(3) / 6 + 0.5
```

Comment:
This is the same as the code of the function below, maybe we can write a function:

```
function assign_c_coefficients(num_stage, cS3)
  c = ones(num_stages) # Best internal stability properties
    c[1] = 0.0

    c[num_stages - 3] = cS3
    c[num_stages - 2] = 0.479274057836310
    c[num_stages - 1] = sqrt(3) / 6 + 0.5
    c[num_stages] = -sqrt(3) / 6 + 0.5
end

```
and call it either here or the constructor.